core/src/main/java/lucee/runtime/config/XMLConfigAdmin.java

```diff
@@ -1665,7 +1665,7 @@ else if (access >= SecurityManager.VALUE_1 && access <= SecurityManager.VALUE_10
 		if (!StringUtil.isEmpty(timezone)) el.setAttribute("timezone", timezone);
 		el.setAttribute("database", database);
 		if (port > -1) el.setAttribute("port", Caster.toString(port));
-		if (connectionLimit > -1) el.setAttribute("connectionLimit", Caster.toString(connectionLimit));
+		el.setAttribute("connectionLimit", Caster.toString(connectionLimit));
```

Comment:
in code i revisit with this, i do `else el.removeAttribute("connectionLimit")`, but i think it is fine,because with Lucee 6 this completely change anyway.